### ISSUE TITLE
React 16.x support

### ISSUE THREAD

**Issue by m2mathew** [justinmc/react-audio-player#42]
Are there plans to push the next version of this package to support React 16.x? The audio players are still working fine for me in our app, but I am getting several peer dependency warnings when doing a `yarn` install. 

Thanks!

**Comment by justinmc** [comment]
I think I just got it working with 16.x!  Make sure you're using the latest release right now ([v0.6.4](https://github.com/justinmc/react-audio-player/releases/tag/v0.6.4)) and try it out.

I cleaned up the peer dependencies a bit and now specify React >=15.  I also separated the example  a little more so that it includes react-audio-player via npm like a real app.  The peer dependency warnings seem to be gone based on running that example now, but let me know if that's not the case for you or if you see any room for improvements.
